### PR TITLE
Added AMSR2 support for LWP validation 

### PR DESCRIPTION
**Pull request by danielphilipp**
Hi,

as already mentioned in the issues section I wanted to add AMSR2 support for LWP validation we need in ESA Cloud_cci+. I first tried to implement the simpler solution to just adapt the reader and then proceed with the existing AMSR-E processing chain. I tested the changed and they seem to work fine. In this PR I wanted to show you more concrete what I wanted to do.

What is your opninion on that ?

Cheers,
Daniel

PS: Because the CCI integration PR #4 is not merged yet, these commits are also included in this PR.

### CODE REVIEW

**Comment by danielphilipp** [comment]
I just found that there are 2 possible data sources for AMSR2 L2B LWP. First one are JAXA retrieved data (which I implemented first), second there are NSIDC retrieved data. Both seem to differ quite significantly concerning the LWP values. Thus I added a reader for both data sources which could be distinguished with the config file switch `AMSR_SENSOR=AMSR2_JAXA` or `AMSR_SENSOR=AMSR2_NSIDC` . For `AMSR-E AMSR_SENSOR=AMSR-E` should be used

File: atrain_match/truths/amsr.py [hunk 1]
```diff
@@ -32,7 +32,34 @@
...
+    return rof
+
+
+def get_amsr_rof(SETTINGS, imager):
+    """ Get radius of influence as a function of overlap and imager pixel size. """
+    if SETTINGS['AMSR_IMAGER_RADIUS'] < 0:
+        if imager == 'seviri':
+            # pixel size 4.8 x 3 km
+            imager_radius = 5660.
+        elif imager == 'avhrr':
```
Comment (ninahakansson) [inline atrain_match/truths/amsr.py:54]: We also match AVHRR for 1km AVHRR_data

Comment (danielphilipp) [reply atrain_match/truths/amsr.py]: Is there an attribute in `cloudproducts` to distinguish them ?

Comment (danielphilipp) [reply atrain_match/truths/amsr.py]: Solved with 66bcbbf

**Comment by ninahakansson** [comment]
Is this ready to merge?

**Comment by danielphilipp** [comment]
Yes should be ready

**Pull request merged**